zipline-kotlin-plugin/src/test/java/app/cash/zipline/kotlin/ZiplineTestInternals.java

```diff
@@ -56,13 +56,13 @@
       JvmClassMappingKt.getKotlinClass(List.class),
       singletonList(KTypeProjection.invariant(stringKt)), false, emptyList());
 
-  public static Pair<KtBridge, KtBridge> newKtBridgePair() {
-    return app.cash.zipline.testing.BridgesKt.newKtBridgePair(new TestCoroutineDispatcher());
+  public static Pair<Endpoint, Endpoint> newEndpointPair() {
```

Comment:
Just curious why this went from Bridge to Endpoint?

Reply:
Did the functionality change?

Reply:
Same exact functionality. I want to call it Endpoint because I want to steal the word `Bridge` for a different symbol in the public API.